test/actions.js

```diff
@@ -643,7 +643,10 @@ module.exports = (app, template, hook) => {
         webhookHandler = (body, url) => {
           body = hook.alter('webhookBody', body);
 
-          assert.equal(body, '');
```

Comment:
We don't need a special flag for this.. I would just remove this line all together.